utilities/mobile_server_pool.py

```diff
@@ -115,9 +165,12 @@ def cleanup_for_blocked_nodes():
         job_id = doc["username"]
         if job_id == "":
             job_id = "test_job:123"
-        job_name, build_id = job_id.split(":")
```

Comment:
this is correct and fast ,we do not need to refactor it.

Reply:
this is not refactoring, this is according to new logic while reserving nodes from server team. This current logic is only expecting the name as job:build, but cannot handle if username has more than that.  When we reserver nodes, we should add other names to username and it fails to get job and build name.

Reply:
You mean we may get more values separated by colon ?